svg/coordinate-systems/inheritance.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>Inheritance of SVG vector-effect</title>
```

Comment:
This file should have a more specific file name. (currently, it's  svg/coordinate-systems/inheritance.svg)

Reply:
Alternately, the title could change. I've been adding inheritance.html for each CSS spec that defines any properties, and inheritance.svg for each SVG chapter that defines any properties.

Reply:
That works. It looks like there aren't any other CSS properties defined in [this chapter](https://svgwg.org/svg2-draft/coords.html), anyway, so that's all that you'd need to do.